### PR TITLE
Windows

### PR DESCRIPTION
**Pull request by scopatz**
This modifies the feedstock to build on windows

### CODE REVIEW

**Comment by jakirkham** [comment]
FTR am fine with this happening. That said, there is ` m2w64-make`, which might work for you for now.

**Comment by scopatz** [comment]
Thanks for letting me know @jakirkham. I didn't know about that package.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,11 +13,17 @@ source:
 
 build:
   number: 0
-  skip: true  # [win]
+
+requirements:  
+  build:
+  - m2w64-toolchain   # [win]
```
Comment (jakirkham) [inline recipe/meta.yaml]: Looks there was a minor indentation issue.

Comment (scopatz) [reply recipe/meta.yaml]: Ahh good catch. fix incoming.

**Pull request merged**